app/src/layouts/instanceLayout/instanceSidebar/instanceSidebar.jsx

```diff
@@ -114,7 +120,7 @@ export const InstanceSidebar = ({ onClickNavBtn }) => {
       }),
     );
 
-    return sidebarItems;
+    return noAssignedOrganizations ? [] : sidebarItems;
```

Comment:
I discussed this with BA, let's remove this condition so that all users can see the page they are currently on.